handout/http/README.md

```diff
@@ -0,0 +1,27 @@
+## HTTP
+
+In order to start making HTTP calls from our Angular app we need to import the `angular/http` module and register for HTTP services. It supports both XHR and JSONP requests exposed through the `HttpModule` and `JsonpModule` respectively. In this section we will be focusing only on the `HttpModule`.
+
+
+### Setting up angular/http
+
+In order to use the various HTTP services we need to include `HttpModule` in the imports for the root `NgModule`. This will allow us to access HTTP services from anywhere in the application.
+
+```ts
+...
+import { MyApp } from './app.component'
+import { HttpModule } from '@angular/http';
+
+@NgModule({
+  imports: [
+    BrowserModule,
+    ReactiveFormsModule,
+    FormsModule,
+    HttpModule
+  ],
+  providers: [SearchService],
+  declarations: [MyApp],
+  bootstrap: [MyApp]
+})
+export class MyAppModule {}
+```
```

Comment:
In some cases we've had commenters complain about the `...` because they think it's destructuring or code. 

I'm not strongly opinionated either way, other than comments potentially making more work.  That said a "busy" example can also make more work in the long run...